pincer/objects/guild/guild.py

```diff
@@ -414,6 +414,9 @@ class Guild(APIObject):
     afk_channel_id: Optional[Snowflake]
     afk_timeout: int
     application_id: Optional[Snowflake]
+    application_command_counts: Optional[Dict]

```

Comment:
@Sigmanificient I think we should use the `APINullable` here with a default of `MISSING` not?

Reply:
Indeed, i believe

Reply:
Updated the code as suggested.